sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -24,17 +28,23 @@ class PrecisionRecallDisplay(_BinaryClassifierCurveDisplayMixin):
 
     Parameters
     ----------
-    precision : ndarray
-        Precision values.
+    precision : ndarray or list of ndarray
+        Precision values. When plotting multiple precision-recall curves, `precision`
+        and `recall` should be lists of the same length.
 
-    recall : ndarray
-        Recall values.
+    recall : ndarray or list of ndarray
+        Recall values. When plotting multiple precision-recall curves, `precision`
+        and `recall` should be lists of the same length.
 
-    average_precision : float, default=None
-        Average precision. If None, the average precision is not shown.
+    average_precision : float or list of floats, default=None
+        Average precision.  When plotting multiple precision-recall curves, can be
+        a list of the same length as `precision` and `recall`.
+        If None, no average precision score is shown.
 
-    estimator_name : str, default=None
-        Name of estimator. If None, then the estimator name is not shown.
+    curve_name : str or list of str, default=None
```

Comment:
I thought `curve_name` is a more generalizable term (vs `estimator_name`), especially with cv multi curves where we want to name each curve by the fold number.

Changing this name will mean that we must change `_validate_plot_params` and thus all other classes that use `_BinaryClassifierCurveDisplayMixin`

I note that the parameter is named differently here (`PrecisionRecallDisplay` init) vs in the `from_prediction` and `from_estimator` methods (where it's called `name`). I'm not sure if this was accidental or to distinguish it from the method parameter 'name's?